client/src/Backend/GameLogic/GameManager.ts

```diff
@@ -4937,17 +4938,15 @@ class GameManager extends EventEmitter {
       toActiveArtifact?.artifactType === ArtifactType.Wormhole &&
       toActiveArtifact.linkTo === fromPlanet.locationId
     ) {
-      if (greaterRarity === undefined) {
-        greaterRarity = toActiveArtifact.rarity;
-      } else {
-        greaterRarity = Math.max(greaterRarity, toActiveArtifact.rarity) as ArtifactRarity;
-      }
+      greaterRarity = greaterRarity === undefined
+        ? toActiveArtifact.rarity
+        : Math.max(greaterRarity, toActiveArtifact.rarity) as ArtifactRarity;
```

Comment:
Not that much a fan of nested loops that makes me think twice, easier with a ternary here `?`.